tests/pipeline/nodes/model/movenetv1/test_movenet.py

```diff
@@ -71,41 +71,33 @@ def test_no_human_single(self, no_human_image, movenet_config_single):
         model = Node(movenet_config_single)
         output = model.run({"img": no_human_img})
         expected_output = {
-            "bboxes": np.zeros(0),
-            "keypoints": np.zeros(0),
-            "keypoint_scores": np.zeros(0),
-            "keypoint_conns": np.zeros(0),
-            "bbox_labels": np.zeros(0),
+            "bboxes": np.empty((0, 4)),
+            "keypoints": np.empty(0),
+            "keypoint_scores": np.empty(0),
+            "keypoint_conns": np.empty(0),
+            "bbox_labels": np.empty(0),
         }
         assert output.keys() == expected_output.keys()
         for i in expected_output.keys():
             npt.assert_array_equal(
-                x=output[i],
-                y=expected_output[i],
-                err_msg=(
-                    f"unexpected output for {i}, Expected {np.zeros(0)} got {output[i]}"
-                ),
+                output[i], expected_output[i], f"unexpected output for `{i}`"
```

Comment:
The original error message was designed to show the expected "ground truth" values vs the obtained values, intention is to be clear up front as to the error, without the developer having to delve into the code.
This change seems to remove the original output behavior.

Reply:
`numpy.testing` provides some information about the content of the actual and expected result which seem to be duplicated by the `Expected {np.zeros(0)} got {output[i]}` portion of `err_msg`.

Error output using `f"unexpected output for {i}, Expected {np.zeros(0)} got {output[i]}"`
```
E           AssertionError: 
E           Arrays are not equal
E           unexpected output for bboxes, Expected [] got [1.]
E           (shapes (1,), (0, 4) mismatch)
E            x: array([1.])
E            y: array([], shape=(0, 4), dtype=float64)

tests/pipeline/nodes/model/movenetv1/test_movenet.py:107: AssertionError
```
The content of the arrays are printed as `x` and `y`. The important part of `err_msg` was showing which key of the output dict has the error as we are iterating through it.

Error message using `f"unexpected output for {i}"`
```
E           AssertionError: 
E           Arrays are not equal
E           unexpected output for `bboxes`
E           (shapes (1,), (0, 4) mismatch)
E            x: array([1.])
E            y: array([], shape=(0, 4), dtype=float64)

tests/pipeline/nodes/model/movenetv1/test_movenet.py:101: AssertionError
```

Specific to the development of this PR, the original `err_msg` was not able to provide the necessary information to help with debugging
```
E           AssertionError: 
E           Arrays are not equal
E           unexpected output for bboxes, Expected [] got []
E           (shapes (0,), (0, 4) mismatch)
E            x: array([], dtype=float64)
E            y: array([], shape=(0, 4), dtype=float64)

tests/pipeline/nodes/model/movenetv1/test_movenet.py:107: AssertionError
```